src/Resources/Resources/help/Get-AzRoleAssignment.md

```diff
@@ -203,6 +203,21 @@ Get-AzRoleAssignment -Scope "/subscriptions/96231a05-34ce-4eb4-aa6a-70759cbb5e83
 
 Gets role assignments at the 'site1' website scope.
 
+### Example 5
+```powershell
+$ApplicationObject = Get-AzADServicePrincipal -DisplayName 'ApplicationDisplayName'
+Get-AzRoleAssignment -ServicePrincipalName $ApplicationObject.AppId
+```
+
+Gets role assignments for the specified Service Principal using Get-AzAdServicePrincipal.
+
+### Example 6
```

Comment:
Example 6 and Example 5 are the same. Can you remove the Example 6?

Reply:
I've removed example 6, I agree it's very similiar, however example 6 was providing a more "easy to read" example for any novice in PowerShell that might not be familiar with objects.

Thanks!